proto/protovalidate/buf/validate/validate.proto

```diff
@@ -190,6 +190,7 @@ message FieldRules {
   //   - proto2 scalar fields (both optional and required)
   // - proto3 scalar fields must be non-zero to be considered populated
   // - repeated and map fields must be non-empty to be considered populated
+  // - map keys/values and repeated items are always considered populated
```

Comment:
Should we also call this out on `RepeatedRules.items`, `MapRules.keys`, `MapRules.values`?

For example:

>   // Note that repeated items are always considered populated. The `required`
>   // rule does not apply.

Reply:
Wouldn't hurt. We're also likely to expand the doc site entry for required and ignore to give way more examples around these to help clarify.

Reply:
FYI, the reference section of the doc site is directly generated from the comments in validate.proto. For example [required](https://buf.build/docs/reference/protovalidate/rules/field_rules/#ignore), and [repeated.items](https://buf.build/docs/reference/protovalidate/rules/repeated_rules/#items).

Reply:
Added here: https://github.com/bufbuild/protovalidate/pull/360/commits/322fb211d502e2d6cf41fe0d6c6ca4d33810f226